site/app/configs/DataHeaderTitle.js

```diff
@@ -0,0 +1,3 @@
+export default {
```

Comment:
Do these have to be separate config files?  Or could we create a combined config that contains all of this type of metadata information?  Definitely not a blocker, it could be future work if it would be of interest.

Reply:
Yeah, I was wondering about that too - I could do that under this ticket(s)